packages/toolpad-studio/src/canvas/index.tsx

```diff
@@ -152,6 +151,15 @@ export default function AppCanvas({ basename, state: initialState }: AppCanvasPr
       return { nodes };
     });
 
+    setCommandHandler(bridge.canvasCommands, 'scrollComponent', (nodeId) => {
+      if (!nodeId) {
+        return;
+      }
+      invariant(appRootRef.current, 'App root not found');
+      const canvasNode = appRootRef.current.querySelector(`[data-node-id='${nodeId}']`);
+      canvasNode?.scrollIntoView({ behavior: 'smooth', block: 'end', inline: 'end' });
```

Comment:
I don't think animation works well for this use case

```suggestion
      canvasNode?.scrollIntoView({ behavior: 'instant', block: 'end', inline: 'end' });
```

Reply:
Done, doing `instant` only.